4-projects/modules/infra_pipelines/variables.tf

```diff
@@ -0,0 +1,75 @@
+/**
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+variable "cloudbuild_project_id" {
+  description = "The project id where the pipelines and repos should be created"
+  type        = string
+}
+
+variable "billing_account" {
+  description = "The ID of the billing account to associated this project with"
+  type        = string
+}
+
+variable "app_infra_repos" {
+  description = "A list of Cloud Source Repos to be created to hold app infra Terraform configs"
+  type        = list(string)
+}
+
+variable "bucket_region" {
+  description = "Region to create GCS buckets for tfstate and Cloud Build artifacts"
+  type        = string
+  default     = "us-central1"
+}
+
+variable "terraform_apply_branches" {
+  description = "List of git branches configured to run terraform apply Cloud Build trigger. All other branches will run plan by default."
+  type        = list(string)
+
+  default = ["development",
+    "non\\-production", //non-production needs a \ to ensure regex matches correct branches.
```

Comment:
I believe we can remove these escape characters from both here and bootstrap. (would need to test, but would be cleaner)

Reply:
Won't this be an issue https://github.com/terraform-google-modules/terraform-google-bootstrap/issues/52

Reply:
(I think this is a relic of using funkier regexes, rather than the invert option)

Reply:
I tested and looks like we can remove the escape chars